internal/jstest/jstest.go

```diff
@@ -53,27 +55,56 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 		TestResults: make(map[string]internal.TestResult),
 	}
 
+	type item struct {
+		file string
+		err  error
+		out  []byte
+	}
+
 	volume := "tests"
-	for _, testName := range files {
-		output, err := runCommand(dir, "mongo", filepath.Join(volume, testName))
-		if err != nil {
-			if _, ok := err.(*exec.ExitError); !ok {
-				return nil, err
+
+	ch := make(chan *item, len(files))
+
+	var wg sync.WaitGroup
+	for _, f := range files {
+		wg.Add(1)
+
+		go func(f string) {
+			defer wg.Done()
+
+			it := &item{
+				file: f,
 			}
-		}
+			it.out, it.err = runCommand(dir, "mongo", filepath.Join(volume, f))
+			ch <- it
+		}(f)
+	}
 
-		if err == nil {
-			res.TestResults[testName] = internal.TestResult{
-				Status: internal.Pass,
-				Output: string(output),
+	// closer
+	go func() {
+		wg.Wait()
+		close(ch)
+	}()
```

Comment:
```suggestion
	wg.Wait()

	close(ch)
```

No need to use a goroutine there

Reply:
Nice!